apps/modernization-ui/src/apps/deduplication/patient-merge/landing/MergeLanding.tsx

```diff
@@ -4,16 +4,30 @@ import { Icon } from 'design-system/icon';
 import { MatchesRequiringReviewTable } from './matches-requiring-review-table/MatchesRequiringReviewTable';
 import { useExportMatches } from 'apps/deduplication/api/useExportMatches';
 import styles from './merge-landing.module.scss';
+import { SortingProvider, useSorting } from 'sorting';
+import { PaginationProvider } from 'pagination';
 
 export const MergeLanding = () => {
+    return (
+        <PaginationProvider pageSize={20}>
+            <SortingProvider>
+                <MergeLandingContent />
+            </SortingProvider>
+        </PaginationProvider>
+    );
+};
+
+export const MergeLandingContent = () => {
```

Comment:
Issue: Remove `export` from `MergeLandingContent`, this should not be imported anywhere.